src/front/app/features/create/components/validate.js

```diff
@@ -0,0 +1,19 @@
+const validate = (values) => {
+    const errors = {};
```

Comment:
It is better to make errors an array. Each field can have more than one validation faille.